test/suites/dev/moonbeam/test-precompile/test-precompile-assets-erc20-local-assets-removal.ts

```diff
@@ -0,0 +1,101 @@
+import { beforeAll, deployCreateCompiledContract, describeSuite, expect } from "@moonwall/cli";
+import { ALITH_ADDRESS } from "@moonwall/util";
+import { Abi } from "viem";
+
+describeSuite({
+  id: "D2584",
```

Comment:
Create a unique ID for this new suite (as it's first test in first suite should be 0101) with a unique letter prefix